config/spec/testnet.go

```diff
@@ -20,24 +20,19 @@
 
 package spec
 
-import (
-	"github.com/berachain/beacon-kit/chain"
-	"github.com/berachain/beacon-kit/primitives/common"
-)
+import "github.com/berachain/beacon-kit/chain"
 
 // TestnetChainSpecData is the chain.SpecData for Berachain's public testnet.
-//
-// TODO: adjust values before testnet genesis.
 func TestnetChainSpecData() *chain.SpecData {
 	specData := MainnetChainSpecData()
+
+	// Testnet uses chain ID of 80069.
 	specData.DepositEth1ChainID = TestnetEth1ChainID
 
-	// Unlike mainnet, testnet activates Bera minting at block 1
-	specData.EVMInflationAddressDeneb1 = common.NewExecutionAddressFromHex(mainnetEVMInflationAddressDeneb1)
-	specData.EVMInflationAddressGenesis = common.NewExecutionAddressFromHex(mainnetEVMInflationAddressDeneb1)
+	// Genesis values of EVM inflation are consistent with those of mainnet.
+	// Testnet activates Deneb1 for BERA minting at epoch 1.
+	specData.Deneb1ForkEpoch = 1
 
-	specData.EVMInflationPerBlockDeneb1 = mainnetEVMInflationPerBlockDeneb1
-	specData.EVMInflationPerBlockGenesis = mainnetEVMInflationPerBlockDeneb1
```

Comment:
as discussed with Rez, removing assignment of values that are already consistent with mainnet spec. Allows the differences to be clearly highlighted here (which are only _when_ the fork epochs are occurring).